Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 #include "I_BatteryStateOfChargeService.h"
-
+#include "BatteryData.h"
```

Comment:
I don't think this needs to be added

Reply:
it does

Reply:
However, having an include in a header slows down compile times a lot, place it in the .cpp file!